tox.ini

```diff
@@ -0,0 +1,27 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py36, py37, py38
+
+[testenv]
+deps =
+  absl-py
+  google-api-core
+  google-cloud-spanner >= 1.17.1
+  frozendict
+  portpicker
```

Comment:
Is there any way to keep this in sync with other places? E.g. could we put this in a `requirements.txt` or something, and reference that here?

Reply:
While these can be put in a `requirements.txt` file, I don't think `setup.py` should be referencing that file, so there still would be duplication
https://stackoverflow.com/a/33685899

Reply:
I ended up adding the below in a different PR because of the same issue with setup.py not referencing requirements.txt. Would it make sense to add requirements.txt here, and use it both for tox and the github workflow? Or would the github workflow use tox and not need to reference the dependencies explicitly anymore?